app/models/iiif_harvester.rb

```diff
@@ -17,6 +17,12 @@ def id
     manifest['@id']
   end
 
+  def collection
+    Settings.vatican.collections.select do |collection|
+      naked_shelfmark.start_with? collection
+    end.max_by(&:length)
```

Comment:
Do you think it would be worth throwing in a unit test for this (just because it has a bit of logic here w/ the `max_by` and `collect`/`start_with?`.

I see we have an integration test for this, so I would happily be ignored.

Reply:
Fixed in a434701.